lib/SetupCheck/HighPerformanceBackend.php

```diff
@@ -34,7 +34,10 @@ public function getName(): string {
 
 	public function run(): SetupResult {
 		if ($this->talkConfig->getSignalingMode() === Config::SIGNALING_INTERNAL) {
-			return SetupResult::success();
+			return SetupResult::error(
+				$this->l->t('No High-performance backend configured - Running Nextcloud Talk without the High-performance backend only scales for very small calls (max. 2-3 participants). Please set up the High-performance backend to ensure calls with multiple participants work seamlessly.'),
+				'https://portal.nextcloud.com/article/Nextcloud-Talk/High-Performance-Backend/Installation-of-Nextcloud-Talk-High-Performance-Backend',
```

Comment:
Doesn't it affect community installations?

Reply:
I couldn't find a better documentation page atm, most importantly is also to get the UI in, so the strings get translated before the next release. We can change the URL later before the release.